alchemy-web/docs/providers/vercel/project.md

```diff
@@ -0,0 +1,60 @@
+# Project
+
+Create and manage Vercel projects.
+
+## Authentication
+
+To use this resource, set `VERCEL_ACCESS_TOKEN` in your `.env` file or pass `accessToken` directly in your resource configuration.
+
+## Examples
+
+### With `accessToken`
+
+```ts
+const project = await Project("my-app", {
+  name: "my-app",
+  framework: "astro",
+  accessToken: alchemy.secret(process.env.VERCEL_ACCESS_TOKEN)
+});
+```
+
+### With GitHub
+
+```typescript
+const project = await Project("my-app", {
+  name: "my-app",
+  framework: "nextjs",
+  gitRepository: {
```

Comment:
nit: `repository`? Or is it important to distinguish git and non-git?

Reply:
I'm hesitant to deviate too much because they also have `repository` in their response object.  Same for `environment`, `projectEnv` and lots of (what I assume) tech debt =/

Reply:
Makes sense, alignment is good.